src/sentry/api/endpoints/organization_member_team_details.py

```diff
@@ -67,6 +76,21 @@ def _can_access(self, request, member, organization):
 
         return False
 
+    def _can_admin_team(self, request, organization, team_slug):
+        try:
+            OrganizationMember.objects.get(
```

Comment:
you can do this as `return OrganizationMember.objects.filter(...).exists()`. Slightly faster because then we don't need to bring the rows into memory.